test/corpus/literals.txt

```diff
@@ -266,528 +250,518 @@ var x = $"{{";
         type: (predefined_type)
         (variable_declarator
           name: (identifier)
-          (equals_value_clause
-            (string_literal
-              (escape_sequence)
-              (string_literal_fragment)
-              (escape_sequence)
-              (string_literal_fragment)
-              (escape_sequence)))))))
+          (string_literal
+            (escape_sequence)
+            (string_literal_content)
+            (escape_sequence)
+            (string_literal_content)
+            (escape_sequence))))))
   (global_statement
     (local_declaration_statement
       (variable_declaration
         type: (predefined_type)
         (variable_declarator
           name: (identifier)
-          (equals_value_clause
-            (verbatim_string_literal))))))
+          (verbatim_string_literal)))))
   (comment)
   (global_statement
     (local_declaration_statement
       (variable_declaration
         type: (implicit_type)
         (variable_declarator
           name: (identifier)
-          (equals_value_clause
-            (string_literal
-              (string_literal_fragment)
-              (string_literal_encoding)))))))
+          (string_literal
+            (string_literal_content)
+            (string_literal_encoding))))))
   (global_statement
     (local_declaration_statement
       (variable_declaration
         type: (implicit_type)
         (variable_declarator
           name: (identifier)
-          (equals_value_clause
-            (string_literal
-              (string_literal_fragment)
-              (string_literal_encoding)))))))
+          (string_literal
+            (string_literal_content)
+            (string_literal_encoding))))))
   (global_statement
     (local_declaration_statement
       (variable_declaration
         type: (implicit_type)
         (variable_declarator
           name: (identifier)
-          (equals_value_clause
-            (verbatim_string_literal))))))
+          (verbatim_string_literal)))))
   (global_statement
     (local_declaration_statement
       (variable_declaration
         type: (implicit_type)
         (variable_declarator
           name: (identifier)
-          (equals_value_clause
-            (verbatim_string_literal))))))
+          (verbatim_string_literal)))))
   (global_statement
     (local_declaration_statement
       (variable_declaration
         type: (predefined_type)
         (variable_declarator
           name: (identifier)
-          (equals_value_clause
-            (interpolated_string_expression
-              (interpolated_string_text)
-              (interpolation
-                (identifier))
-              (interpolated_string_text)))))))
+          (interpolated_string_expression
+            (interpolation_start)
+            (string_content)
+            (interpolation
+              (interpolation_open_brace)
+              (identifier)
+              (interpolation_close_brace))
+            (string_content))))))
```

Comment:
Why do we include the start, quote and brace tokens in the tree?

Reply:
they can be variable in size, i could hide them but this then becomes unqueryable, or i could alias them to a literal, e.g. alias open_brace to '{', but that wouldn't always be correct since it could be multiple

Reply:
Makes sense. I'd keep it as it is. It just seemed unusual to see them in the tree.